Python/Product/Cookiecutter/ViewModel/TemplateViewModel.cs

```diff
@@ -0,0 +1,179 @@
+                string name;
+                ParseRepoOwnerAndName(RemoteUrl, out owner, out name);
+                return owner;
+            }
+        }
+
+        /// <summary>
+        /// Repository full name, ie. 'owner/name'.
+        /// </summary>
+        public string RepositoryFullName {
+            get {
+                string owner;
+                string name;
+                ParseRepoOwnerAndName(RemoteUrl, out owner, out name);
+                if (!string.IsNullOrEmpty(owner) && !string.IsNullOrEmpty(name)) {
+                    return owner + '/' + name;
+                }
+                return string.Empty;
+            }
+        }
+
+        public string DisplayName {
+            get {
+                return _displayName;
+            }
+
+            set {
+                if (value != _displayName) {
+                    _displayName = value;
+                    PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(nameof(DisplayName)));
+                }
+            }
+        }
+
+        public string RemoteUrl {
+            get {
+                return _remoteUrl;
+            }
+
+            set {
+                if (value != _remoteUrl) {
+                    _remoteUrl = value;
+                    PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(nameof(RemoteUrl)));
+                }
+            }
+        }
+
+        public string ClonedPath {
+            get {
+                return _clonedPath;
+            }
+
+            set {
+                if (value != _clonedPath) {
+                    _clonedPath = value;
+                    PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(nameof(ClonedPath)));
+                }
+            }
+        }
+
+        public string Description {
+            get {
+                return _description;
+            }
+
+            set {
+                if (value != _description) {
+                    _description = value;
+                    PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(nameof(Description)));
+                }
+            }
+        }
+
+        public bool IsSearchTerm {
+            get {
+                return _isSearchTerm;
+            }
+
+            set {
+                if (value != _isSearchTerm) {
+                    _isSearchTerm = value;
+                    PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(nameof(IsSearchTerm)));
+                }
+            }
+        }
+
+        public ImageMoniker Image {
+            get {
+                return _image;
+            }
+
+            set {
+                if (value.Guid != _image.Guid || value.Id != _image.Id) {
+                    _image = value;
+                    PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(nameof(Image)));
+                }
+            }
+        }
+
+        private static bool ParseRepoOwnerAndName(string repoUrl, out string owner, out string name) {
```

Comment:
>ParseRepoOwnerAndName [](start = 28, length = 21)

I'd rather see this done with a regex:

var m = Regex.Match(@"http(s)?://github\.com/(?P<owner>.+?)/(?P<name>.+?)(/|$)", repoUrl);
owner = m.Groups["owner"].Value;
name = m.Groups["name"].Value;
return m.Groups["owner"].Success && m.Group["name"].Success;